x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings_by_resource/resource_findings/resource_findings_container.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { EuiSpacer, EuiButtonEmpty } from '@elastic/eui';
+import type { DataView } from '@kbn/data-plugin/common';
+import { Link, useParams } from 'react-router-dom';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useEuiTheme } from '@elastic/eui';
+import { generatePath } from 'react-router-dom';
+import * as TEST_SUBJECTS from '../../test_subjects';
+import { PageWrapper, PageTitle, PageTitleText } from '../../layout/findings_layout';
+import { useCspBreadcrumbs } from '../../../../common/navigation/use_csp_breadcrumbs';
+import { findingsNavigation } from '../../../../common/navigation/constants';
+
+const BackToResourcesButton = () => {
+  return (
+    <Link to={generatePath(findingsNavigation.findings_by_resource.path)}>
```

Comment:
can we make this remember the page number you were in?

Reply:
currently it is out of scope of this task

Reply:
not sure i follow, but generally - there are no page numbers in `group-by-resource` as we fetch based on `after_key`, not an index

Reply:
I thought @JordanSh meant to save the state (in the url or other way)
but I see I might be wrong